api/Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM node:16
+FROM node:lts-alpine
 
 WORKDIR /usr/src/app
 
 COPY package*.json ./
 
-RUN npm install --unsafe-perm=true --allow-root --legacy-peer-deps
+RUN yarn install
 
 COPY . .
 
 RUN chmod 777 node_modules
 
+RUN yarn build
+
 ARG DB_URL=DB_URL
 ARG PORT=PORT
 ARG NODE_ENV=NODE_ENV
 
 EXPOSE 8080/tcp
 
-CMD ["node", "server.js"]
\ No newline at end of file
+CMD ["node", "dist/server.js"]
```

Comment:
do we compile the code to a dist folder in prod?

Reply:
Everything happens in docker for prod. The the yarn build  in the docker builds the ts to js.